modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once the 30% limit is reached the container is killed and the gathering process stops. Information already gathered will be downloaded to your local storage. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
+
+If you reach the storage limit you will see a message output informing you what happened.
+
+For example:
+
+[source,terminal]
+----
+Disk usage exceeds the volume percentage of %d for mounted directory. Exiting...
+----
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
```

Comment:
I don't see this assembly used in the OSD or ROSA topic maps (it's commented out both places). So I can't check this ifdef - so maybe we should just leave it out?